src/commands.jl

```diff
@@ -233,6 +233,7 @@ function maybe_step_through_wrapper!(@nospecialize(recurse), frame::Frame)
         # this is a wrapper function that we might want to step through
         while frame.pc != length(stmts)-1
```

Comment:
```suggestion
        while frame.pc < length(stmts)-1
```

Reply:
Hm, seems like this broke the tests. Thoughts?

Reply:
I'll just keep the current for now to get this in and then we can think about the `<` :P